gitopsaddon/README.md

```
@@ -1109,6 +1112,45 @@ If the OLM operator deployment never appears in `openshift-gitops-operator` afte
    KUBECONFIG=/path/to/ocp kubectl get installplan -n openshift-gitops-operator
    ```
 
+### ArgoCD UI Live Manifest Shows "Resource not found in cluster"
+
+This error has two common causes in agent mode:
+
+**Cause 1: Destination-Based Mapping mismatch between principal and agent**
+
+The `destinationBasedMapping` setting controls the Redis key separator (principal uses `_`, agent without DBM uses `|`). If the principal has DBM enabled but the agent does not, the agent logs show:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)